src/components/Export.tsx

```diff
@@ -170,10 +176,65 @@ const ExportDialog: ExportDialogComponent = ({
         }
       }
     };
-
     const pageKey = getPageKey(tldraw);
 
-    if (typeof results !== "object") return;
+    type TLdrawProps = {
+      [key: string]: any;
+    };
+    type ShapeBounds = {
+      x: number;
+      y: number;
+      w: number;
+      h: number;
+    };
+    const extractShapesBounds = (tldraw: TLdrawProps): ShapeBounds[] => {
+      return Object.keys(tldraw)
+        .filter((key) => tldraw[key].typeName === "shape")
+        .map((key) => {
+          const shape = tldraw[key];
+          return {
+            x: shape.x,
+            y: shape.y,
+            w: shape.props.w,
+            h: shape.props.h,
+          };
+        });
+    };
+    const shapeBounds = extractShapesBounds(tldraw);
+
+    type CommonBounds = {
+      top: number;
+      right: number;
+      bottom: number;
+      left: number;
+    };
+    const findCommonBounds = (shapes: ShapeBounds[]): CommonBounds => {
```

Comment:
surprised tldraw doesnt expose something for this

Reply:
They do, `app.allShapesCommonBounds` but I don't know how to get the `app` (the tlddraw instance of a canvas that isn't loaded) in `Export.tsx`.  

Did a little digging and found `useEditor`, but I believe you can only use that in children of the tldraw component.

Reply:
oh damn. is this method just a copy paste of what they're doing in there?

Reply:
Nah, I figured it was easy enough to make so just did it.